README.md

```diff
@@ -242,6 +243,7 @@ _Mailing lists servers._
 _Mass mailing software - one message to many recipients_
 
   * [phpList](https://phplist.org) - Newsletter and email marketing with advanced management of subscribers, bounces, and plugins. ([Source Code](https://github.com/phpList/)) `AGPLv3` `PHP`
+   * [Mailtrain](https://mailtrain.org/) - self hosted newsletter application built on Node.js (v5+) and MySQL (v5.5+ or MariaDB). ([Source Code](https://github.com/andris9/mailtrain)) `GPL-V3.0` `Node.js`
```

Comment:
Fixed. Just noticed it. I guess my terminal generates one extra space (zsh)! Sorry for the trouble